ibm_was/tests/test_unit.py

```diff
@@ -40,6 +40,11 @@ def test_custom_query(aggregator, instance, check):
         'ibm_was.object_pool.objects_created_count',
         'implementations:ObjectPool_ibm.system.objectpool_com.ibm.ws.webcontainer.srt.SRTConnectionContextImpl',
     )
+    # Prove that we already collect the thread_pools.percent_used metric, satisfying AI-2672.
+    aggregator.assert_metric(
```

Comment:
why is this in test_custom_query test?

Reply:
ooops, I originally had a separate test for the `thread_pools.percent_used` metric, then thought we could merge the two, but forgot to update the name.

Will change that!

Reply:
I actually decided to put it into a separate test after all.

Reply:
would it make sense to add it to METRICS_ALWAYS_PRESENT instead? Also I do not think it makes sense for the in-code comment to mention the jira ticket

Reply:
Yup, this should work, done!

Reply:
We can cycle back to this later if you want, I'll merge this for now so that we can try it out on staging.